pypeit/core/wavecal/echelle.py

```diff
@@ -0,0 +1,213 @@
+
```

Comment:
Add a short note on what this module is for

Reply:
It is already there: Module for echelle-specific wavelength calibration functions.